### ISSUE TITLE
Multiple zones are silently ignored without --geo

### ISSUE THREAD

**Issue by benesch** [cockroachdb/roachprod#110]
`roachprod create --zones=one,two` should throw an error instead of silently selecting the first zone.

**Comment by bobvawter** [comment]
Or should manually specifying multiple zones force `--geo`?

**Comment by benesch** [comment]
Yeah, that works too! But then it should be an error to specify `--geo --zones=one,two`, because it's not clear which option should apply.
